docs/functional_testing.md

```diff
@@ -106,7 +106,7 @@ See [Local Testing](../README.md#local-testing) for instructions to set up your
 ### Ensure third SP is forced to authenticate
 * Browse to http://ssp-sp3.local:8083/module.php/core/authenticate.php?as=ssp-hub
 * This should redirect to http://ssp-idp1.local:8085/module.php/core/loginuserpass.php?AuthState=...
-* Login as admin using "a" as the password.
+* Login as sildisco_idp1 using "sildisco_password" as the password (without the quotation marks).
```

Comment:
is this correct or should it be 3?

Reply:
It's correct. The password is for IdP1 not SP3